static/app/views/issueDetails/groupSidebar.tsx

```diff
@@ -258,6 +259,14 @@ export default function GroupSidebar({
 
   return (
     <Container>
+      {((organization.features.includes('ai-summary') &&
+        issueTypeConfig.issueSummary.enabled) ||
+        issueTypeConfig.resources) && (
```

Comment:
Should we be checking for Autofix separately too?

Reply:
I thought we don't support Autofix if we don't have a summary? We haven't designed an entry point for that case anyway.